lib/streamlit/elements/streamlit_plotly_theme.py

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import plotly.graph_objects as go
+import plotly.io as pio
+
+# This is the streamlit theme for plotly where we pass in a template.data
+# and a template.layout.
+
+# Template.data is for changing specific graph properties in a general aspect
+# such as Contour plots or Waterfall plots.
+
+# Template.layout is for changing things such as the x axis and fonts and other
+# general layout properties for general graphs.
+
+# We pass in temporary colors to the frontend and the frontend will replace
+# those colors because we want to change colors based on the background color.
+
+# Start at #0000001 because developers may be likely to use #000000
+CATEGORY_0 = "#000001"
```

Comment:
nit: can you do a quick check via code search in the plotly repo if any of the black color variations we use here is used anywhere within plotly already? Just to make sure that we are not causing some unexpected issues.

Reply:
Nothing seen